### PR TITLE
add pib command for basic block level disassembly

### PR DESCRIPTION
**Pull request by kamou**
this PR is related to #7453

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -3317,7 +3317,7 @@ static int cmd_print(void *data, const char *input) {
 // r_cons_printf ("|Usage: pi[defj] [num]\n");
 		{
 			const char *help_msg[] = {
-				"Usage:", "pi[defrj] [num]", "",
+				"Usage:", "pi[defbrj] [num]", "",
```
Comment (radare) [inline libr/core/cmd_print.c]: the alphabetical order is intentional

**Comment by radare** [comment]
can you add a test too? Thanks!

File: libr/core/cmd_print.c (hunk 1 above)
Comment (kamou) [inline libr/core/cmd_print.c]: oops ok !

**Comment by kamou** [comment]
just give me some time to figure out how to add a test :)

**Pull request merged**